### ISSUE TITLE
Support error middleware

### ISSUE THREAD

**Issue by knownasilya** [izaakschroeder/express-context#11]
`function (err, res, req, next)` signature.

**Comment by izaakschroeder** [comment]
Good point. You can't (yet) contextualize error middleware, since e.g. https://github.com/izaakschroeder/express-context/blob/master/lib/contextualize.js#L84 never checks for the 4 argument signature. I can add support for this though.
